.github/workflows/_push_image.yml

```diff
@@ -0,0 +1,50 @@
+name: Push Image
+
+on:
+  workflow_call:
+    inputs:
+      repo_type:
+        description: "Public or Private Repository"
+        required: true
+        type: string
+      image_tag:
+        description: Tag to push the image with
+        required: true
+        type: string
+      image:
+        description: Image to Push (e.g. primary:path/to/Dockerfile)
+        required: true
+        type: string
+      docker_context:
+        description: Docker Context to Use
+        required: true
+        type: string
+
+jobs:
+  push-example-image:
+    name: Push Image (${{ inputs.image }})
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Log in to Azure Container Registry
+        uses: azure/docker-login@v1.0.1
+        with:
+          login-server: ${{ secrets.AZURE_REGISTRY_URL }}
+          username: ${{ secrets.AZURE_REGISTRY_USERNAME }}
+          password: ${{ secrets.AZURE_REGISTRY_PASSWORD }}
+
+      - name: Get Linux Executables
+        if: ${{ contains(inputs.image, 'helloworld') }}
+        run: pushd hello-world && chmod +x get_verbose_report.sh && ./get_verbose_report.sh && popd
+
+      - name: Build and Push Docker Image
+        uses: docker/build-push-action@v5
+        env:
+          DOCKERFILE_CONTEXT: ${{ inputs.docker_context }}
```

Comment:
I don't see where the `DOCKERFILE_CONTEXT` env is used. also, since `inputs.docker_context` is used in the `with` block right below, is the `DOCKERFILE_CONTEXT` env needed?
```  
with:
    context: ${{ inputs.docker_context }}
 ```

Reply:
Good catch, another thing from copy-paste lol